packages/@roots/bud-api/src/methods/persist.ts

```diff
@@ -27,51 +26,33 @@ declare module '@roots/bud-framework' {
 
 const persist: Api.Persist = function (enabled = true) {
   isEqual(enabled, true) &&
-    (() => {
-      /**
-       * Cache override
-       */
-      this.hooks
-        .on('build/cache', () => ({
-          name: this.hooks.filter('build/cache/name'),
-          type: this.hooks.filter('build/cache/type'),
-          version: this.hooks.filter('build/cache/version'),
-          cacheDirectory: this.hooks.filter(
-            'build/cache/cacheDirectory',
-          ),
-          cacheLocation: this.hooks.filter(
-            'build/cache/cacheLocation',
-          ),
-          managedPaths: this.hooks.filter(
-            'build/cache/managedPaths',
-          ),
-          buildDependencies: this.hooks.filter(
-            'build/cache/buildDependencies',
-          ),
-        }))
+    this.hooks
+      .on('build/cache', () => ({
+        type: this.hooks.filter('build/cache/type'),
+        version: this.hooks.filter('build/cache/version'),
+        cacheDirectory: this.hooks.filter(
+          'build/cache/cacheDirectory',
+        ),
+        managedPaths: this.hooks.filter(
+          'build/cache/managedPaths',
+        ),
+        buildDependencies: this.hooks.filter(
+          'build/cache/buildDependencies',
+        ),
+      }))
 
-        /**
-         * Individual settings
-         */
-        .hooks.on('build/cache/name', () => this.cache.cacheName)
-        .hooks.on(
-          'build/cache/version',
-          () => this.cache.version,
-        )
-        .hooks.on('build/cache/type', () => 'filesystem')
-        .hooks.on('build/cache/cacheDirectory', () =>
-          this.path('storage'),
-        )
-        .hooks.on('build/cache/cacheLocation', () =>
```

Comment:
Letting webpack handle this internally for a similar reason https://github.com/roots/bud/pull/64/files#r637445882